test/e2e-v2/java-test-service/Dockerfile.consumer

```diff
@@ -0,0 +1,27 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
I think we can now directly use the agent image and mount the service jar in docker compose file, so we can only change that single file if we want to update the agent image

Reply:
If you only want to update the agent image in one place, please take a look, It have been define in the GHA file: https://github.com/apache/skywalking/pull/7810/files#diff-29a4ad0b26179d51ca486039a03d45f783f02a5906e198292001358a29673c2fR44-R45

Reply:
> If you only want to update the agent image in one place, please take a look, It have been define in the GHA file: https://github.com/apache/skywalking/pull/7810/files#diff-29a4ad0b26179d51ca486039a03d45f783f02a5906e198292001358a29673c2fR44-R45

What I want is update it in one place, and take effect both locally and in GHA.

Reply:
The current codes will make it different in local tests and in GHA. If we update the GHA env var and forgot to update the default value.

Reply:
Ok, It makes sense. I will remove this docker file and declare it in the docker-compose file.

Reply:
But for the satellite, we also need to use the dockerfile to define it. because we need to using `nc` to check health. The satellite is using scratch as the base image.

Reply:
Could we move all version information into an initiative script? Neither GHA nor docker are not controlling it?

Reply:
One file managing all test related 3rd party resources are what I expect.

Reply:
Yes, we could. But I think this script need to export all environment as a local file(because the environment has scope), GHA and locally using `source` to import them.

Reply:
I am not good about bash, @kezhenxu94 any suggestion?

Reply:
> But for the satellite, we also need to use the dockerfile to define it. because we need to using `nc` to check health. The satellite is using scratch as the base image.

The `ARG` can also be specified in `docker-compose.yaml` file. What you need to do is just remove the default value in `Dockerfile.satellite` and set it in `docker-compose.yaml`, so we can manage all versions in the `docker-compose.yaml`

```yaml
  satellite:
    build:
      context: ../
      dockerfile: Dockerfile.satellite
      args:
        - SW_AGENT_SATELLITE_COMMIT=${SW_AGENT_SATELLITE_COMMIT}
```

Reply:
Yes, It's like this now. but we have multiple docker-compose files, some in the base-compose, some in the other docker-compose file. So we need someplace to put all the config as one file.

Reply:
@mrproliu OK, then please create a single file to manage the versions. Maybe we can just export the versions in `setup` steps ?

P.S. I think we need to support the `.env` file feature in infra-e2e, which seems to be a more standard means in docker-compose to customize the variables.

Reply:
> @mrproliu OK, then please create a single file to manage the versions. Maybe we can just export the versions in `setup` steps ?

Currently, the `setup.step` is running after the compose setup is finished. Also, we need to make sure the environment in `.env` is available when the compose setup(there also have scope issue, but it could be resolved).